app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -100,7 +105,7 @@ module.exports = React.createClass
       @handleFocus()
 
```

Comment:
Maybe set the dropdown value on `componentWillReceiveProps` as well as on mount? There's something weird going on where the dropdown gets its value from the annotation property, so property changes need to override component state. I think I've got my head round it now.